.circleci/config.yml

```diff
@@ -405,6 +405,9 @@ jobs:
             if [ "${CIRCLE_BRANCH}" != "doc-prod" ]; then
               pip uninstall -y plotly
               cd ../packages/python/plotly
+              # To be removed after plotly.js release
+              pip install inflect black tox
+              python3 setup.py updateplotlyjsdev --devbranch fast-image
```

Comment:
my idea here was to be able to take a look at the doc even when the corresponding feature is not released in plotly.js, but it does not work since the exported html asks for the regular plotly.js bundle so the feature is not available. Too bad! Or maybe I can find a way to link to another bundle...